components/automate-ui/src/app/entities/projects/project.reducer.ts

```diff
@@ -21,16 +32,26 @@ const CREATE_STATUS = 'createStatus';
 const CREATE_ERROR = 'createError';
 const DELETE_STATUS = 'deleteStatus';
 const UPDATE_STATUS = 'updateStatus';
+const APPLY_RULES_STATUS = 'applyRulesStatus';
 
 export const projectEntityAdapter: EntityAdapter<Project> = createEntityAdapter<Project>();
 
+const initialApplyRulesStatus: ApplyRulesStatus = {
+  state: 'not_running',
+  estimatedTimeComplete: '0001-01-01T00:00:00Z',
+  percentageComplete: 61,
```

Comment:
why 61?

Reply:
Ha oops, just had that there while I was testing some stuff out w/ the progress bar. I'll change it back to 0.